### DISCUSSION TITLE
I wonder what causes the printing error

### DISCUSSION THREAD

**Discussion by kitcoun** [MicrosoftEdge/WebView2Feedback#4022] [Q&A]
OtherError will appear when using print method without prompt box. No problem with other methods

Print Settings

CoreWebView2PrintSettings printSettings = null;
printSettings = webView21.CoreWebView2.Environment.CreatePrintSettings();
printSettings.PrinterName = printerName;
return printSettings;

**Comment by novac42** [comment]
Thanks for reporting the issue. Could you please provide some more information about this?

	• Framework (WinUI, Win32, WinForms, UWP, WPF, etc)
	• SDK and Runtime version.
	• Operating System
This will help with figuring out how to best address your issue. Thanks!

**Comment by monica-ch** [comment]
@kitcoun Do you have any issues printing to the same printer using print dialog (ctrl+p or print from context menu) from the webview2 app?

Also, could you please provide more details regarding framework, SDK and runtime version and operating system.
